client/components/ItemCard.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const ItemCard = ({ item }) => {
+  return (
+    <div className='itemCard'>
```

Comment:
in some other files, i see that your classnames are camelcase, some are like item-card (idk the formal name for this)
be consistent! for css selectors, the convention is item-card